src/utils/index.ts

```diff
@@ -43,3 +43,5 @@ export const extractHashAndKeyFromVSShareUrl = (
   const hash = parser.match(urlWithPathParam).hash
   return { hash, key }
 }
+
+export const JSON_SCHEMA_URL = 'https://schema.affinidi.com/EventElegibilityV1-0.json'
```

Comment:
```suggestion
export const JSON_SCHEMA_URL = 'https://schema.affinidi.com/EventEligibilityV1-0.json'
```

Reply:
we created new schema for new fields in issuing form.
So, `Eligibility` is the new one